.github/workflows/test.yml

```diff
@@ -1,12 +1,8 @@
 name: Test
 
 on:
-  # disabled before linting PR comes in
-  # push:
-  #   branches:
-  #     - master
-  #     - develop
-  # pull_request:
+  push:
```

Comment:
what do you think about running the tests on pull requests as well?

Reply:
Would be good to have it, but think there are two issues with running it:
1. API limits on Etherscan and Infura, if same API key is used by master and all the forks, it could timeout for some tests if they run at the same time? 
2. If these tests then fail, to have them running again and finishing will take some time, so instead of PRs being light-weight it might mean it needs an hour or so to finish tests.

Happy to add it back, but just these 2 cents. 😄